bcgov/mds
# MDS-4196: Create Project namespace to support IRT development

# Main

- Ticket: https://bcmines.atlassian.net/jira/software/c/projects/MDS/boards/1?modal=detail&selectedIssue=MDS-4196
- Please see ticket for refactor details

# Other

- Project_summary_title, proponent_project_id, project_summary_lead_party_guid have been removed from project_summary and placed at the project level
- Project_summary_contact is not currently in use but still exists in the API, this will be addressed as more Project requirements become available

# How to test

- Verify Project Descriptions CRUD still works in Minespace
- Verify Project Descriptions RU still works in Core

# Notes

- UI is still currently the same, changes will be coming in a upcoming sprint
